packages/http/src/webauthn.ts

```diff
@@ -131,7 +136,27 @@ export async function getWebAuthnAssertion(
 export async function getWebAuthnAttestation(
   options: TurnkeyCredentialCreationOptions
 ): Promise<TAttestation> {
+  const webAuthnSupported = await hasWebAuthnSupport();
+
+  if (!webAuthnSupported) {
+    throw new Error("webauthn is not supported by this browser");
+  }
+
   const res = await webauthnCredentialCreate(options);
 
   return toInternalAttestation(res.toJSON());
 }
+
+// For additional details see https://web.dev/articles/passkey-form-autofill#feature-detection, https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI
+async function hasWebAuthnSupport(): Promise<boolean> {
+  if (
+    window.PublicKeyCredential &&
+    PublicKeyCredential.isConditionalMediationAvailable
+  ) {
+    const isCMA = await PublicKeyCredential.isConditionalMediationAvailable();
+
+    return isCMA;
+  }
```

Comment:
I think this is too restrictive? `isConditionalMediationAvailable` is specific to the autofill feature, which isn't needed for barebone webauthn/passkey support. In our case it's probably enough to return `window.PublicKeyCredential !== undefined`?